package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "socket.io",
```

Comment:
This change makes `package-lock.json` to contain a stable root package name (otherwise it uses the directory name)